### PR TITLE
feat(addresses): add company to addresses and removes checkPresent

### PR DESCRIPTION
**Pull request by dmlittle**
What: 
- Adds `company` support to AddressRequest and AddressResponse
- Removes checkPresent on name

@pon

### CODE REVIEW

**Comment by javadev** [comment]
LGTM :)

**Comment by javadev** [comment]
!

**Comment by pon** [comment]
@dmlittle - looks like a handful of errors on the build

**Comment by dmlittle** [comment]
@pon errors where caused by the latest API version changes. I've rebased this off lob/lob-java#58 which fixes those issues. We need to make sure to merge this **after**.

File: src/main/java/com/lob/protocol/request/AddressRequest.java [hunk 1]
```diff
@@ -8,17 +8,18 @@
 import java.util.Map;
 
 import static com.lob.Util.checkNotNull;
-import static com.lob.Util.checkPresent;
 
 public class AddressRequest extends AbstractAddressRequest implements HasLobParams {
     private final static int MAX_LENGTH = 50;
 
     private final String name;
+    private final String company;
```
Comment (robinjoseph08) [inline src/main/java/com/lob/protocol/request/AddressRequest.java]: address also needs description

Comment (robinjoseph08) [inline src/main/java/com/lob/protocol/request/AddressRequest.java]: or is this out of the scope of this PR?

Comment (javadev) [inline src/main/java/com/lob/protocol/request/AddressRequest.java]: `description` field is defined inside AbstractLobRequest.

**Comment by javadev** [comment]
!

**Pull request merged**